platform/linux-dpdk/Makefile.am

```diff
@@ -196,6 +197,7 @@ noinst_HEADERS = \
 		  ${top_srcdir}/platform/linux-generic/include/odp_packet_socket.h \
 		  ${top_srcdir}/platform/linux-generic/include/odp_pkt_queue_internal.h \
 		  ${srcdir}/include/odp_pool_internal.h \
```

Comment:
I am thinking if the file name needs to be changed to indicate that this is a pool implemented using DPDK. Currently, linux-generic mempool also has odp_pool_internal.h (this also should to be renamed to indicate 'generic' pool)